py/torch_tensorrt/_Input.py

```diff
@@ -337,18 +337,18 @@ def from_tensor(
             A Input object.
         """
         if not (
-            t.is_contiguous(memory_format=torch.contiguous_format)
+            disable_memory_format_check
+            or t.is_contiguous(memory_format=torch.contiguous_format)
```

Comment:
Certain 0D tensors do not have the attribute `is_contiguous`, specifically when fake-ified. This logic reordering preserves the original conditional expression but ensures we do not access that attribute when `disable_memory_format_check` is set.